AgentDiscoveries-Frontend/app/src/components/admin/entities-view.jsx

```diff
@@ -47,6 +47,11 @@ export default class Entities extends React.Component {
         // Only rendering the table if there is an entity.
         // In this case, use the first to extract the header labels
         const entity = this.state.entities[0];
+
+        if(this.props.api == "users"){ //user specific operations
```

Comment:
This feels like an odd place to do this to me - we've got user specific logic in a general table, and we rely on anything in the front end ignoring the password field.

It feels to me like it would be better if we avoided serializing the passwords in the first place (but we still allow them to deserialize).

We could consider the 'exclusion strategies' mentioned in [this article](https://www.baeldung.com/gson-exclude-fields-serialization)?